MonstieWash/Assets/Scripts/Erasing/Tasks/TaskTracker.cs

```diff
@@ -5,17 +5,25 @@
 
```

Comment:
```suggestion
[RequireComponent(typeof(RoomSaver))]
[RequireComponent(typeof(UIManager))]
[RequireComponent(typeof(SoundPlayer))]
```

Reply:
As the number of components required by the class increases some error protection could be useful to stop those pesky designers